src/model/CategoricalColumn.ts

```diff
@@ -127,11 +127,11 @@ export default class CategoricalColumn extends ValueColumn<string> implements IC
     //TODO infer categories from data
 
     this.setRendererList([
-      {type: 'categorical', label: 'Default'},
-      {type: 'catcolor', label: 'Category Color'},
-      {type: 'upset', label: 'UpSet'}
+      {type: 'categorical', label: 'Color + Label'},
+      {type: 'catcolor', label: 'Color'},
+      {type: 'upset', label: 'Matrix'}
```

Comment:
why is upset no longer upset?

Reply:
@mstreit ?

Reply:
because it's not a real upset. @alex also mentioned this in one of our previous meetings.

Reply:
it is neither a matrix. And if the column would have multiple categories it would have shown a line between them like in Upset